develop/app/Database/Seeds/Product.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Database\Seeds;
+
+use CodeIgniter\Database\Seeder;
+use Faker\Factory;
+
+class Product extends Seeder
+{
+    public function run()
+    {
+        $faker = Factory::create();
+        $now   = date("Y-m-d H:i:s");
+
+        for ($i = 0; $i < 20; $i++) {
+            $data = [
+                'name'       => $faker->words(1, true),
```

Comment:
If my memory services me right, the faker library provide the fake produce data can let user use.
Try yo use the real product data.

Reply:
This chahges is done.